Content.Server/Radiation/Systems/RadiationSystem.GridCast.cs

```diff
@@ -211,4 +226,18 @@ private void UpdateGridcast()
 
         return ray;
     }
+
+    private float ApplyModForRadiationBlockingContainers(EntityUid uid, float rads)
```

Comment:
just name this GetAdjustedRadStrength or something more generic. In the future we might have multiple entities that affect how strong the radiation on an object is.

Reply:
Ok, 'tis done